lib/ops/opsservice/install.go

```diff
@@ -897,7 +890,7 @@ func (s *site) newProvisioningToken(operation ops.SiteOperation) (token string,
 		tokenRequest.Expires = s.clock().UtcNow().Add(defaults.InstallTokenTTL)
 	}
 	_, err = s.users().CreateProvisioningToken(tokenRequest)
-	if err != nil {
+	if err != nil && !trace.IsAlreadyExists(err) {
```

Comment:
Why ignore this here too? Shouldn't the caller be making this decision?

Reply:
Yes, moved to caller but in general all these `IsAlreadyExists` checks are mitigating the non-transactional nature of the backend operation.